src/exercises/day-1/book-library.rs

```diff
@@ -116,6 +118,9 @@ fn main() {
     //}
     //
     //println!("Our library has {} books", library.len());
+    for book in library.books {
+        println!("{book}");
+    }
```

Comment:
I just realized that this inadvertently shows the students what they are supposed to write in the `print_books` method. I'll fix this for the class I'm teaching next week.